.circleci/config.yml

```diff
@@ -141,6 +141,10 @@ run_flake8: &run_flake8
         flake8 --show-source --statistics
 
 
+# TODO (Min): figure out how to do coverage nightly or on-demand. Doing it
```

Comment:
+1 I did a bit of digging regarding this last month. We can definitely do nightly coverage. The first step is to figure out which tests should be nightly and which are per commit.